src/vstart.sh

```diff
@@ -1004,14 +1004,20 @@ do_rgw()
         RGWDEBUG="--debug-rgw=20"
     fi
 
+    CEPH_RGW_HTTPS="${CEPH_RGW_PORT: -1}"
+    if [[ ${CEPH_RGW_HTTPS} = "s" ]]; then
+        CEPH_RGW_PORT="${CEPH_RGW_PORT::-1}"
+    else
+        CEPH_RGW_HTTPS=""
+    fi
     RGWSUDO=
     [ $CEPH_RGW_PORT -lt 1024 ] && RGWSUDO=sudo
     n=$(($CEPH_NUM_RGW - 1))
     i=0
     for rgw in j k l m n o p q r s t u v; do
-	echo start rgw on http://localhost:$((CEPH_RGW_PORT + i))
-	run 'rgw' $RGWSUDO $CEPH_BIN/radosgw -c $conf_fn --log-file=${CEPH_OUT_DIR}/rgw.$rgw.log ${RGWDEBUG} --debug-ms=1 -n client.rgw "--rgw_frontends=${rgw_frontend} port=$((CEPH_RGW_PORT + i))"
-	i=$(($i + 1))
+        echo start rgw on http://localhost:$((CEPH_RGW_PORT + i))${CEPH_RGW_HTTPS}
```

Comment:
it's better to replace `http` to `http${CEPH_RGW_HTTPS}`

Reply:
yes, will change to 
``` bash
echo start rgw on http${CEPH_RGW_HTTPS}://localhost:$((CEPH_RGW_PORT + i))
```
so that the displayed URL can be copy-pasted